src/Tracking/Conversions.php

```diff
@@ -0,0 +1,200 @@
+				'custom_data' => array(
+					'currency'    => $data->get_currency(),
+					'value'       => $data->get_price() * $data->get_quantity(),
+					'content_ids' => array( $data->get_id() ),
+					'contents'    => array(
+						array(
+							'id'         => $data->get_id(),
+							'item_price' => $data->get_price(),
+							'quantity'   => $data->get_quantity(),
+						),
+					),
+					'num_items'   => $data->get_quantity(),
+				),
+			);
+		}
+
+		if ( Tracking::EVENT_VIEW_CATEGORY === $event_name ) {
+			/** @var Category $data */
+			$data = array(
+				'event_id'    => $data->get_event_id(),
+				'custom_data' => array(
+					'category_name' => $data->getName(),
+				),
+			);
+		}
+
+		if ( Tracking::EVENT_ADD_TO_CART === $event_name ) {
+			/** @var Product $data */
+			$data = array(
+				'event_id'    => $data->get_event_id(),
+				'custom_data' => array(
+					'currency'    => $data->get_currency(),
+					'value'       => $data->get_price() * $data->get_quantity(),
+					'content_ids' => array( $data->get_id() ),
+					'contents'    => array(
+						array(
+							'id'         => $data->get_id(),
+							'item_price' => $data->get_price(),
+							'quantity'   => $data->get_quantity(),
+						),
+					),
+					'num_items'   => $data->get_quantity(),
+				),
+			);
+		}
+
+		if ( Tracking::EVENT_CHECKOUT === $event_name ) {
+			/** @var Checkout $data */
+			$data = array(
+				'event_id'    => $data->get_event_id(),
+				'custom_data' => array(
+					'currency'    => $data->get_currency(),
+					'value'       => $data->get_price() * $data->get_quantity(),
+					'content_ids' => array_map(
+						function ( Product $product ) {
+							return $product->get_id();
+						},
+						$data->get_items()
+					),
+					'contents'    => array_map(
+						function ( Product $product ) {
+							return array(
+								'id'         => $product->get_id(),
+								'item_price' => $product->get_price(),
+								'quantity'   => $product->get_quantity(),
+							);
+						},
+						$data->get_items()
+					),
+					'num_items'   => $data->get_quantity(),
+				),
+			);
+		}
+
+		if ( $data instanceof None ) {
+			$data = array(
+				'event_id' => $data->get_event_id(),
+			);
+		}
+
+		$ad_account_id = Pinterest_For_WooCommerce()::get_setting( 'tracking_advertiser' );
+		$event_name    = static::EVENT_MAP[ $event_name ] ?? '';
+
+		/** @var array $data */
+		$data = array_merge(
+			$data,
+			array(
+				'event_name'       => $event_name,
+				'action_source'    => 'web',
+				'event_time'       => time(),
+				'event_source_url' => home_url( $wp->request ),
+				'partner_name'     => 'ss-woocommerce',
+				'user_data'        => array(
+					'client_ip_address' => $this->user->get_client_ip_address(),
+					'client_user_agent' => $this->user->get_client_user_agent(),
+				),
+				'language'         => 'en',
+			)
+		);
+
+		try {
+			/* Translators: 1: Conversions API event name, 2: JSON encoded event data. */
```

Comment:
```suggestion
			/* translators: 1: Conversions API event name, 2: JSON encoded event data. */
```

The format expects `translators` starting with the lower case.